app/src/main/java/com/github/sdpteam15/polyevents/database/Database.kt

```diff
@@ -1,16 +1,19 @@
 package com.github.sdpteam15.polyevents.database
 
+import android.util.Log
+
 /**
  * Database
  */
 object Database {
-    private var mutableCurrentDatabase: DatabaseInterface? = null
+    var mutableCurrentDatabase: DatabaseInterface? = null
     var currentDatabase: DatabaseInterface
         get() {
             mutableCurrentDatabase = mutableCurrentDatabase ?: FirestoreDatabaseProvider
             return mutableCurrentDatabase!!
         }
         set(value) {
+            Log.d("Database", "New DB " + value.toString())
```

Comment:
We should remove this when we will merge no ? :)

Reply:
@Rygone sorry Robin I refer to you for this one as well.

Reply:
corrected